From 88c517dd053d6b42cbd8686658da8df524f91bc8 Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Mon, 21 May 2012 20:48:35 -0500
Subject: [PATCH 1515/2508] ENGR00210271 - PWM1 pad incorrectly configured by
 EPDC driver

- Remove configuration of PWM1 pad for EPDC.  Was there for debug purposes,
but caused problems with LCD support.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 arch/arm/mach-mx6/board-mx6sl_arm2.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6sl_arm2.h b/arch/arm/mach-mx6/board-mx6sl_arm2.h
index f4cc7fe..940c46d 100755
--- a/arch/arm/mach-mx6/board-mx6sl_arm2.h
+++ b/arch/arm/mach-mx6/board-mx6sl_arm2.h
@@ -182,8 +182,6 @@ static iomux_v3_cfg_t mx6sl_arm2_epdc_enable_pads[] = {
 	MX6SL_PAD_EPDC_PWRSTAT__GPIO_2_13,
 	MX6SL_PAD_EPDC_PWRCTRL0__GPIO_2_7,
 	MX6SL_PAD_EPDC_PWRWAKEUP__GPIO_2_14,
-
-	MX6SL_PAD_PWM1__CCM_CLKO,
 };
 
 static iomux_v3_cfg_t mx6sl_arm2_epdc_disable_pads[] = {
-- 
1.7.7.4

